tutorials/monitoring/2-setting-up-environment/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Monitoring NServiceBus solutions: Setting up monitoring environment"
+reviewed: 2017-10-10
+summary: Setting up the Particular Service Platform to monitor an NServiceBus system.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Configuring endpoints"
+  nextUrl: tutorials/monitoring/3-configuring-endpoints
+---
+
+include: monitoring-intro-paragraph
+
+include: monitoring-sample-solution
+
+This second lesson guides your through setting up all of the server side components that are used to monitor an NServiceBus system.
+
+
+## Setting up
+
+In order to configure an environment for monitoring with the Particular Service Platform you will need to install and configure the components in the order listed.
+
+
+### Install ServiceControl
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system. The main installer includes a desktop utility called the ServiceControl Management Utility which can be used to create and manage ServiceControl and Monitoring instances.
+
+Download and run the latest ServiceControl installer.
+
+SCREENSHOT - ServiceControl Installer
+
+Once it has finished installing, run the ServiceControl Management Utility.
+
+
+### Create ServiceControl instance
+
+A ServiceControl instance collects messages from the error queue and manages retrying them. It also collects messages from the audit queue. 
+
+In the ServiceControl Management Utility, click **+ NEW INSTANCE** and select **ServiceControl instance** from the list of options.
+
+SCREENSHOT - SMCU Main Page with Drop-down opened and Add ServiceControl instance selected
+
+NOTE: Creating and managing ServiceControl instances requires a license file. If you do not already have a license then the ServiceControl Management Utility will generate a trial license for you. If you have already had a trial license and it has expired, then follow the on-screen prompts to extend your trial license. It is a short form and your new license file will get sent to your email address. See (/servicecontrol/license.md) for information about how to install your trial license.
+
+Under TRANSPORT CONFIGURATION select the transport that your NServiceBus system runs on. Depending on the transport you may be required to add an additional connection string.
```

Comment:
FYI my convention with the tutorials thus far (stolen from Packt writing guidelines) is to have all on-screen, non-code text (things you're selecting, clicking, looking for, etc.) all be in bold. So I think that would cover **TRANSPORT CONFIGURATION** here (assume you're using caps because SCMU does?), **SQLServer** in the NOTE below, **AUDIT FORWARDING** + **On** + **Off** in the next paragraph, etc.

It's personally jarring to me to have the caps there. Honestly CONFIGURATION looks misspelled to me all in caps! I had to check it four times. I'd consider doing **Word Case in Bold** in those instances even though SCMU is all-caps.